### PR TITLE
Stopgap fix: prefer docker over podman for kind

### PR DESCRIPTION
**Pull request by maya-r**
We require additional changes to run some CI lanes and jumped too
far ahead by preferring podman if it ever exists.

See https://github.com/kubevirt/project-infra/pull/2212 for example failures.

This makes it possible to have podman as a package on the CI images
without the additional changes being done yet.

### CODE REVIEW

**Comment by maya-r** [comment]
cc @oshoval
Intended to unblock https://github.com/kubevirt/project-infra/pull/2213 once this is merged to kubevirt.
But there might be more changes needed.

**Review by oshoval — APPROVED** [review]
Thanks

**Comment by oshoval** [comment]
In case the bootstrap wont have both podman and docker at the end, please don't merge this PR

**Comment by oshoval** [comment]
/lgtm cancel

Please keep me updated what was decided and will take a look again

**Comment by maya-r** [comment]
I'm not in any secret discussion forums :)

**Comment by oshoval** [comment]
> I'm not in any secret discussion forums :)

Even if you were, it should be a secret :) j/k
Just confused if there are going to be two seperated images as the original plan - in this case this PR is not needed
or some other combination where one of the bootstraps will still have both docker and podman rpm

**Comment by maya-r** [comment]
closing in favour of not requiring podman on the hpp/velero repos -- to be compatible with brian carey's proposal to have a separate podman-only image and switch CI lanes to it one by one.

**Pull request closed without merging**
